node_modules/gh-calendar/lib/util.js

```diff
@@ -37,13 +37,51 @@ var eventsToICal = module.exports.eventsToICal = function(calendarInfo, events)
 
     // Convert each Grasshopper event to an ical VEvent and add it to the calendar
     _.each(events, function(event) {
+        // Get the organisers as simple strings. Ensure they are sorted so the iCal
+        // feed doesn't change between fetches
+        var organisers = _.chain(event.organisers || [])
+            .sort()
+            .map(function(organiser) {
+                if (_.isString(organiser)) {
+                    return organiser;
+                } else {
+                    return organiser.displayName;
+                }
+            })
+            .value();
+
+        // The description of the iCal event depends on the type of application
+        // and how many organisers / speakers there are
+        var description = '';
+        if (calendarInfo.app && calendarInfo.app.type === 'timetable') {
+            if (organisers.length == 1) {
+                description = 'Lecturer: ';
+            } else {
+                description = 'Lecturers: ';
+            }
+        } else {
+            if (organisers.length == 1) {
+                description = 'Speaker: ';
+            } else {
+                description = 'Speakers: ';
+            }
+        }
+        description += organisers.join(', ') + '\n';
```

Comment:
If `event.organisers` is `null`, this will add an empty `Lecturers: ` to the description. This should probably all be wrapped in a `if (event.organisers) {`